static/templyte/src/app/create-template/create-template.component.ts

```diff
@@ -121,22 +123,34 @@ export class CreateTemplateComponent implements OnInit {
                 data: {message: 'Please enter a name for your template.'}
             });
         }
-        if (this.variableArray.length === 0) {
-            returnVal = false;
-            this.dialog.open(InputValidateDialogComponent, {
-                width: '250px',
-                data: {message: 'Please enter at least one variable name for your template.'}
-            });
-        } else {
-            for (let i = 0; i < this.variableArray.length - 1; i++) {
-                for (let j = i + 1; j < this.variableArray.length; j++) {
-                    if (this.variableArray[i].name === this.variableArray[j].name) {
-                        returnVal = false;
-                        this.dialog.open(InputValidateDialogComponent, {
-                            width: '250px',
-                            data: {message: 'Please do not enter duplicate variables'}
-                        });
-                    }
+        for (let i = 0; i < this.variableArray.length - 1; i++) {
+            for (let j = i + 1; j < this.variableArray.length; j++) {
+                if (this.variableArray[i].name === this.variableArray[j].name) {
+                    returnVal = false;
+                    this.dialog.open(InputValidateDialogComponent, {
+                        width: '250px',
```

Comment:
Can we switch the width of these to something more viewport independent (such as something measured in vh)